docs/faqs.md

```diff
@@ -195,3 +195,8 @@ them to be discovered again.
 Another solution is to add a bind in the kubelet deployment configuration to
 mount the parent directory of local volumes into the kubelet container at the
 same path. This requires restarting the kubelet container.
+
+A famous Kubernetes distribution that runs the kubelet in a container is
```

Comment:
simplify the first part of the sentance to something like:

"For Rancher clusters, users must configure ..."

Reply:
updated